tests/test_mongod_binary.py

```diff
@@ -0,0 +1,137 @@
+"""Test pytest_motor.mongod_binary."""
+import tempfile
+from pathlib import Path
+
+from pytest import MonkeyPatch, mark, raises
+
+from pytest_motor.mongod_binary import MongodBinary
+
+# Archives imitate direcotry structure of real ones
+# but do NOT contain real mongod binaries
+# P.S. Linux and MacOS structure is the same
+MONGODB_MACOS_TAR = Path(__file__).parent.parent / "test_data" / \
```

Comment:
Would it be possible to construct these in the tests rather than checking them in?

Reply:
In particular we could use `pyfakefs`.

Reply:
I did it with fixtures and `tempfile` module.